### PR TITLE
Add Bitflyer and Bithumb

### PR DESCRIPTION
**Pull request by spazvt**
add 'Bitflyer' and 'Bithumb' crypto currency exchanges from fork
  https://github.com/acceptbitcoincash/acceptbitcoincash

Bithumb is a major South Korean exchange right now.

### CODE REVIEW

**Comment by spazvt** [comment]
These two websites have A LOT of traffic. 
[Bithumb Alexa rank is 4930](https://www.alexa.com/siteinfo/bithumb.com) globally and 153 in South Korea.
[Bitflyer Alexa rank is 2932](https://www.alexa.com/siteinfo/bitflyer.jp) globally and 236 in Japan.

Please use google translate to verify the [Google Authenticator and OTP instructions for bithumb](http://bithumb.cafe/archives/7286). (I don't know Korean, but it links directly to their user page to show users how to enable it.)

File: _data/cryptocurrencies.yml [hunk 1]
```diff
@@ -44,6 +53,14 @@ websites:
       hardware: Yes
       doc: https://bitgo.zendesk.com/hc/en-us/sections/201009049-Two-Factor-Authentication-Help-Page
 
+    - name: Bithumb
+      url: https://www.bithumb.com/
+      img: bithumb.png
+      tfa: Yes
+      sms: Yes
+      software: Yes
+      doc: http://bithumb.cafe/archives/7286
```
Comment (Carlgo11) [inline _data/cryptocurrencies.yml]: Seems that they support HTTPS (even though some stuff still load HTTP). Could you change the doc to `https://bithumb.cafe/archives/7286 `?

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by spazvt** [comment]
URL updated to https and verified that all of my other open PRs include only https links.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
The bitflyer logo you provided contains text which often makes the logo look weird when shrunken down to 32x32 pixels. Here's one from their site that just is the logo (without any text).
32x32

**Comment by spazvt** [comment]
ok I'll avoid text next time.

**Review by Carlgo11 — APPROVED** [review]

**Review by brycx — APPROVED** [review]

**Pull request merged**